src/sentry/api/serializers/models/project.py

```diff
@@ -42,7 +42,7 @@ def serialize(self, obj, attrs, user):
         from sentry import features
 
         feature_list = []
-        for feature in ('event-types', 'global-events', 'user-reports', 'dsym'):
+        for feature in ('breadcrumbs', 'event-types', 'global-events', 'user-reports', 'dsym'):
```

Comment:
oops, yeah wasnt intending for this to end up here

i'll back this out of this PR and stuff it into the other